.env.example

```diff
@@ -94,5 +48,35 @@ LOCAL_FILE_UPLOAD=1
 
 POSTFIX_SERVER=postfix
 
-## DISABLE_REGISTRATION=1
-## DISABLE_ONBOARDING=true
+# DISABLE_REGISTRATION=1
+# DISABLE_ONBOARDING=true
+
+## TRAEFIK
```

Comment:
Is there a reason why this block should be moved to the end ?

Nitpick: for consistency, I suggest:

```patch
- ## TRAEFIK
+ ## Traefik
```